### PR TITLE
refactor: use lambdas

### PR DESCRIPTION
**Pull request by CAMOBAP**
- tested on emulator API 16

### CODE REVIEW

File: sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java [hunk 1]
```diff
@@ -30,8 +30,8 @@ public HCaptchaWebView(@NonNull Context context, @Nullable AttributeSet attrs, i
     /**
      * Workaround for crash in WebViewChromium
      * Details:
-     *  - https://stackoverflow.com/questions/58519749
-     *  - https://github.com/zulip/zulip-mobile/issues/4051#issuecomment-616855833
+     *  - <a href="https://stackoverflow.com/questions/58519749">...</a>
+     *  - <a href="https://github.com/zulip/zulip-mobile/issues/4051#issuecomment-616855833">...</a>
```
Comment (DSergiu) [inline sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java:34]: Should we add text instead of `...`?

Comment (CAMOBAP) [reply sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java:34]: Good point. But I cannot propose something better than: Issue investigation p1, Issue investigation p2

**Review by DSergiu — APPROVED** [review]

File: sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java (hunk 1 above)
Comment (DSergiu) [reply sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java:34]: Issue 1 and Issue 2 sounds good to me.

Comment (CAMOBAP) [reply sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java:34]: They are both related to the same issue:
 - SO link describes the issue and suggests a workaround
 - Second one is more about conditions/versions that are affected

Comment (DSergiu) [inline sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java:34]:
```suggestion
     *  - <a href="https://stackoverflow.com/questions/58519749">stackoverflow</a>
     *  - <a href="https://github.com/zulip/zulip-mobile/issues/4051#issuecomment-616855833">github issues</a>
```

**Pull request merged**